content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -34,25 +34,26 @@ The first step when adding non-Kubernetes services to an Istio mesh is to config
 generate the configuration files that let mesh expansion VMs connect to the mesh. To prepare the
 cluster for mesh expansion, run the following commands on a machine with cluster admin privileges:
 
-1.  Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion=true` at
+1.  Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion.enabled=true` at
     install time with Helm, there are two options for enabling mesh expansion, depending on how you originally installed
     Istio on the cluster:
 
     *   If you installed Istio with Helm and Tiller, run `helm upgrade` with the new option:
 
     {{< text bash >}}
     $ cd install/kubernetes/helm/istio
-    $ helm upgrade --set global.meshExpansion=true istio-system .
+    $ helm upgrade --set global.meshExpansion.enabled=true istio-system .
     $ cd -
     {{< /text >}}
 
     *   If you installed Istio without Helm and Tiller, use `helm template` to update your configuration with the option and
         reapply with `kubectl`:
 
     {{< text bash >}}
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
```

Comment:
There's a `helm template` way to do this now

Reply:
Do you mean the `helm template` generated YAML will include CRD definition as well? In the 
```
helm template --set global.meshExpansion.enabled=true --namespace istio-system . > istio.yaml
ack -i "CustomResourceDefinition istio.yaml
```

I didn't see istio API are there.

Could you provide some cmd or links?

Reply:
There's an `istio-init` template that installs the CRDs.

https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template